---
# Documentation: https://wowchemy.com/docs/managing-content/

title: Shifts in the evolutionary rate and intensity of purifying selection between
  two Brassica genomes revealed by analyses of orthologous transposons and relics
  of a whole genome triplication
subtitle: ''
summary: ''
authors:
- Meixia Zhao
- Jianchang Du
- Feng Lin
- Chaobo Tong
- Jingyin Yu
- Shunmou Huang
- Xiaowu Wang
- Shengyi Liu
- Jianxin Ma
tags:
- Genome
- Plant
- Evolution
- Molecular
- Gene Duplication
- Epigenesis
- Genetic
- Gene Expression Regulation
- Plant
- transposable elements
- Selection
- Genetic
- Brassica
- Retroelements
- DNA
- Plant
- purifying selection
- asymmetric evolution
- genetic recombination
- nucleotide substitution
- Terminal Repeat Sequences
- transcriptional alteration
categories: []
date: '2013-10-01'
lastmod: 2022-08-19T18:46:18+08:00
featured: false
draft: false

# Featured image
# To use, add an image named `featured.jpg/png` to your page's folder.
# Focal points: Smart, Center, TopLeft, Top, TopRight, Left, Right, BottomLeft, Bottom, BottomRight.
image:
  caption: ''
  focal_point: ''
  preview_only: false

# Projects (optional).
#   Associate this post with one or more of your projects.
#   Simply enter your project's folder or file name without extension.
#   E.g. `projects = ["internal-project"]` references `content/project/deep-learning/index.md`.
#   Otherwise, set `projects = []`.
projects: []
publishDate: '2022-08-19T10:46:18.828497Z'
publication_types:
- '2'
abstract: Recent sequencing of the Brassica rapa and Brassica oleracea genomes revealed
  extremely contrasting genomic features such as the abundance and distribution of
  transposable elements between the two genomes. However, whether and how these structural
  differentiations may have influenced the evolutionary rates of the two genomes since
  their split from a common ancestor are unknown. Here, we investigated and compared
  the rates of nucleotide substitution between two long terminal repeats (LTRs) of
  individual orthologous LTR-retrotransposons, the rates of synonymous and non-synonymous
  substitution among triplicated genes retained in both genomes from a shared whole
  genome triplication event, and the rates of genetic recombination estimated/deduced
  by the comparison of physical and genetic distances along chromosomes and ratios
  of solo LTRs to intact elements. Overall, LTR sequences and genic sequences showed
  more rapid nucleotide substitution in B. rapa than in B. oleracea. Synonymous substitution
  of triplicated genes retained from a shared whole genome triplication was detected
  at higher rates in B. rapa than in B. oleracea. Interestingly, non-synonymous substitution
  was observed at lower rates in the former than in the latter, indicating shifted
  densities of purifying selection between the two genomes. In addition to evolutionary
  asymmetry, orthologous genes differentially regulated and/or disrupted by transposable
  elements between the two genomes were also characterized. Our analyses suggest that
  local genomic and epigenomic features, such as recombination rates and chromatin
  dynamics reshaped by independent proliferation of transposable elements and elimination
  between the two genomes, are perhaps partially the causes and partially the outcomes
  of the observed inter-specific asymmetric evolution.
publication: '*The Plant Journal: For Cell and Molecular Biology*'
doi: 10.1111/tpj.12291
---
