---
# Documentation: https://wowchemy.com/docs/managing-content/

title: Genome-Wide Association Mapping Identifies Novel Loci for Quantitative Resistance
  to Blackleg Disease in Canola
subtitle: ''
summary: ''
authors:
- Harsh Raman
- Brett McVittie
- Ramethaa Pirathiban
- Rosy Raman
- zhangyuanyuan
- Denise M. Barbulescu
- Yu Qiu
- liushengyi
- Brian Cullis
tags:
- linkage disequilibrium
- canola
- genome-wide association analysis
- Leptosphaeria maculans
- natural variation
- resistance to blackleg
categories: []
date: '2020-01-01'
lastmod: 2022-08-19T18:45:25+08:00
featured: false
draft: false

# Featured image
# To use, add an image named `featured.jpg/png` to your page's folder.
# Focal points: Smart, Center, TopLeft, Top, TopRight, Left, Right, BottomLeft, Bottom, BottomRight.
image:
  caption: ''
  focal_point: ''
  preview_only: false

# Projects (optional).
#   Associate this post with one or more of your projects.
#   Simply enter your project's folder or file name without extension.
#   E.g. `projects = ["internal-project"]` references `content/project/deep-learning/index.md`.
#   Otherwise, set `projects = []`.
projects: []
publishDate: '2022-08-19T10:45:25.513315Z'
publication_types:
- '2'
abstract: Blackleg disease, caused by the fungal pathogen Leptosphaeria maculans,
  continues to be a major concern for sustainable production of canola (Brassica napus
  L.) in many parts of the world. The deployment of effective quantitative resistance
  (QR) is recognized as a durable strategy in providing natural defense to pathogens.
  Herein, we uncover loci for resistance to blackleg in a genetically diverse panel
  of canola accessions by exploiting historic recombination events which occurred
  during domestication and selective breeding by genome-wide association analysis
  (GWAS). We found extensive variation in resistance to blackleg at the adult plant
  stage, including for upper canopy infection. Using the linkage disequilibrium and
  genetic relationship estimates from 12,414 high quality SNPs, GWAS identified 59
  statistically significant and \"suggestive\" SNPs on 17 chromosomes of B. napus
  genome that underlie variation in resistance to blackleg, evaluated under field
  and shade-house conditions. Each of the SNP association accounted for up to 25.1%
  of additive genetic variance in resistance among diverse panel of accessions. To
  understand the homology of QR genomic regions with Arabidopsis thaliana genome,
  we searched the synteny between QR regions with 22 ancestral blocks of Brassicaceae.
  Comparative analyses revealed that 25 SNP associations for QR were localized in
  nine ancestral blocks, as a result of genomic rearrangements. We further showed
  that phenological traits such as flowering time, plant height, and maturity confound
  the genetic variation in resistance. Altogether, these findings provided new insights
  on the complex genetic control of the blackleg resistance and further expanded our
  understanding of its genetic architecture.
publication: '*Frontiers in Plant Science*'
doi: 10.3389/fpls.2020.01184
---
