---
# Documentation: https://wowchemy.com/docs/managing-content/

title: Gene Expression Changes During the Allo-/Deallopolyploidization Process of
  Brassica napus
subtitle: ''
summary: ''
authors:
- Qi Pan
- Bin Zhu
- Dawei Zhang
- tongchaobo
- Xianhong Ge
- liushengyi
- Zaiyun Li
tags:
- Brassica napus
- gene expression
- polyploidization
- DNA methylation
- B. rapa
categories: []
date: '2019-01-01'
lastmod: 2022-08-19T18:46:17+08:00
featured: false
draft: false

# Featured image
# To use, add an image named `featured.jpg/png` to your page's folder.
# Focal points: Smart, Center, TopLeft, Top, TopRight, Left, Right, BottomLeft, Bottom, BottomRight.
image:
  caption: ''
  focal_point: ''
  preview_only: false

# Projects (optional).
#   Associate this post with one or more of your projects.
#   Simply enter your project's folder or file name without extension.
#   E.g. `projects = ["internal-project"]` references `content/project/deep-learning/index.md`.
#   Otherwise, set `projects = []`.
projects: []
publishDate: '2022-08-19T10:46:17.270045Z'
publication_types:
- '2'
abstract: Gene expression changes due to allopolyploidization have been extensively
  studied in plants over the past few decades. Nearly all these studies focused on
  comparing the changes before and after genome merger. In this study, we used the
  uniquely restituted Brassica rapa (RBR, AeAe, 2n = 20) obtained from Brassica napus
  (AnAnCnCn, 2n = 38) to analyze the gene expression changes and its potential mechanism
  during the process of allo-/deallopolyploidization. RNA-seq-based transcriptome
  profiling identified a large number of differentially expressed genes (DEGs) between
  RBR and natural B. rapa (ArAr), suggesting potential effects of allopolyploidization/domestication
  of AA component of B. napus at the tetrapolyploid level. Meanwhile, it was revealed
  that up to 20% of gene expressions were immediately altered when compared with those
  in the An-subgenome. Interestingly, one fifth of these changes are in fact indicative
  of the recovery of antecedent gene expression alternations occurring since the origin
  of B. napus and showed association with homoeologous expression bias between An
  and Cn subgenomes. Enrichment of distinct gene ontology (GO) categories of the above
  sets of genes further indicated potential functional cooperation of the An and Cn
  subgenome of B. napus. Whole genome methylation analysis revealed a small number
  of DEGs were identified in the differentially methylated regions.
publication: '*Frontiers in Genetics*'
doi: 10.3389/fgene.2019.01279
---
