---
# Documentation: https://wowchemy.com/docs/managing-content/

title: Genome-wide comparative analysis of 20 miniature inverted-repeat transposable
  element families in Brassica rapa and B. oleracea
subtitle: ''
summary: ''
authors:
- Perumal Sampath
- Jayakodi Murukarthick
- Nur Kholilatul Izzah
- Jonghoon Lee
- Hong-Il Choi
- Kenta Shirasawa
- Beom-Soon Choi
- liushengyi
- Ill-Sup Nou
- Tae-Jin Yang
tags:
- Genome
- Plant
- Genomics
- DNA Transposable Elements
- Base Sequence
- Species Specificity
- Brassica rapa
- Inverted Repeat Sequences
categories: []
date: '2014-01-01'
lastmod: 2022-08-19T18:46:18+08:00
featured: false
draft: false

# Featured image
# To use, add an image named `featured.jpg/png` to your page's folder.
# Focal points: Smart, Center, TopLeft, Top, TopRight, Left, Right, BottomLeft, Bottom, BottomRight.
image:
  caption: ''
  focal_point: ''
  preview_only: false

# Projects (optional).
#   Associate this post with one or more of your projects.
#   Simply enter your project's folder or file name without extension.
#   E.g. `projects = ["internal-project"]` references `content/project/deep-learning/index.md`.
#   Otherwise, set `projects = []`.
projects: []
publishDate: '2022-08-19T10:46:18.349060Z'
publication_types:
- '2'
abstract: Miniature inverted-repeat transposable elements (MITEs) are ubiquitous,
  non-autonomous class II transposable elements. Here, we conducted genome-wide comparative
  analysis of 20 MITE families in B. rapa, B. oleracea, and Arabidopsis thaliana.
  A total of 5894 and 6026 MITE members belonging to the 20 families were found in
  the whole genome pseudo-chromosome sequences of B. rapa and B. oleracea, respectively.
  Meanwhile, only four of the 20 families, comprising 573 members, were identified
  in the Arabidopsis genome, indicating that most of the families were activated in
  the Brassica genus after divergence from Arabidopsis. Copy numbers varied from 4
  to 1459 for each MITE family, and there was up to 6-fold variation between B. rapa
  and B. oleracea. In particular, analysis of intact members showed that whereas eleven
  families were present in similar copy numbers in B. rapa and B. oleracea, nine families
  showed copy number variation ranging from 2- to 16-fold. Four of those families
  (BraSto-3, BraTo-3, 4, 5) were more abundant in B. rapa, and the other five (BraSto-1,
  BraSto-4, BraTo-1, 7 and BraHAT-1) were more abundant in B. oleracea. Overall, 54%
  and 51% of the MITEs resided in or within 2 kb of a gene in the B. rapa and B. oleracea
  genomes, respectively. Notably, 92 MITEs were found within the CDS of annotated
  genes, suggesting that MITEs might play roles in diversification of genes in the
  recently triplicated Brassica genome. MITE insertion polymorphism (MIP) analysis
  of 289 MITE members showed that 52% and 23% were polymorphic at the inter- and intra-species
  levels, respectively, indicating that there has been recent MITE activity in the
  Brassica genome. These recently activated MITE families with abundant MIP will provide
  useful resources for molecular breeding and identification of novel functional genes
  arising from MITE insertion.
publication: '*PloS One*'
doi: 10.1371/journal.pone.0094499
---
