---
# Documentation: https://wowchemy.com/docs/managing-content/

title: The impact and origin of copy number variations in the Oryza species
subtitle: ''
summary: ''
authors:
- baizetao
- Jinfeng Chen
- Yi Liao
- Meijiao Wang
- Rong Liu
- Song Ge
- Rod A. Wing
- Mingsheng Chen
tags:
- Genome
- Plant
- High-Throughput Nucleotide Sequencing
- Sequence Analysis
- DNA
- Oryza
- DNA Copy Number Variations
- Phenotype
- Molecular Sequence Annotation
- DNA
- Plant
- CNV genes
- Copy number variation (CNV)
- Mutation mechanisms
- NGS-based survey
- Oryza species
categories: []
date: '2016-03-01'
lastmod: 2022-08-15T21:04:17+08:00
featured: false
draft: false

# Featured image
# To use, add an image named `featured.jpg/png` to your page's folder.
# Focal points: Smart, Center, TopLeft, Top, TopRight, Left, Right, BottomLeft, Bottom, BottomRight.
image:
  caption: ''
  focal_point: ''
  preview_only: false

# Projects (optional).
#   Associate this post with one or more of your projects.
#   Simply enter your project's folder or file name without extension.
#   E.g. `projects = ["internal-project"]` references `content/project/deep-learning/index.md`.
#   Otherwise, set `projects = []`.
projects: []
publishDate: '2022-08-15T13:04:17.559380Z'
publication_types:
- '2'
abstract: "BACKGROUND: Copy number variation (CNV), a complex genomic rearrangement,\
  \ has been extensively studied in humans and other organisms. In plants, CNVs of\
  \ several genes were found to be responsible for various important traits; however,\
  \ the cause and consequence of CNVs remains largely unknown. Recently released next-generation\
  \ sequencing (NGS) data provide an opportunity for a genome-wide study of CNVs in\
  \ rice. RESULTS: Here, by an NGS-based approach, we generated a CNV map comprising\
  \ 9,196 deletions compared to the reference genome 'Nipponbare'. Using Oryza glaberrima\
  \ as the outgroup, 80% of the CNV events turned out to be insertions in Nipponbare.\
  \ There were 2,806 annotated genes affected by these CNV events. We experimentally\
  \ validated 28 functional CNV genes including OsMADS56, BPH14, OsDCL2b and OsMADS30,\
  \ implying that CNVs might have contributed to phenotypic variations in rice. Most\
  \ CNV genes were found to be located in non-co-linear positions by comparison to\
  \ O. glaberrima. One of the origins of these non-co-linear genes was genomic duplications\
  \ caused by transposon activity or double-strand break repair. Comprehensive analysis\
  \ of mutation mechanisms suggested an abundance of CNVs formed by non-homologous\
  \ end-joining and mobile element insertion. CONCLUSIONS: This study showed the impact\
  \ and origin of copy number variations in rice on a genomic scale."
publication: '*BMC genomics*'
doi: 10.1186/s12864-016-2589-2
---
