---
# Documentation: https://wowchemy.com/docs/managing-content/

title: An improved allele-specific PCR primer design method for SNP marker analysis
  and its application
subtitle: ''
summary: ''
authors:
- Jing Liu
- Shunmou Huang
- Meiyu Sun
- liushengyi
- Yumei Liu
- Wanxing Wang
- Xiurong Zhang
- Hanzhong Wang
- Wei Hua
tags: []
categories: []
date: '2012-08-01'
lastmod: 2022-08-19T18:46:19+08:00
featured: false
draft: false

# Featured image
# To use, add an image named `featured.jpg/png` to your page's folder.
# Focal points: Smart, Center, TopLeft, Top, TopRight, Left, Right, BottomLeft, Bottom, BottomRight.
image:
  caption: ''
  focal_point: ''
  preview_only: false

# Projects (optional).
#   Associate this post with one or more of your projects.
#   Simply enter your project's folder or file name without extension.
#   E.g. `projects = ["internal-project"]` references `content/project/deep-learning/index.md`.
#   Otherwise, set `projects = []`.
projects: []
publishDate: '2022-08-19T10:46:19.113487Z'
publication_types:
- '2'
abstract: "BACKGROUND: Although Single Nucleotide Polymorphism (SNP) marker is an\
  \ invaluable tool for positional cloning, association study and evolutionary analysis,\
  \ low SNP detection efficiency by Allele-Specific PCR (AS-PCR) still restricts its\
  \ application as molecular marker like other markers such as Simple Sequence Repeat\
  \ (SSR). To overcome this problem, primers with a single nucleotide artificial mismatch\
  \ introduced within the three bases closest to the 3'end (SNP site) have been used\
  \ in AS-PCR. However, for one SNP site, nine possible mismatches can be generated\
  \ among the three bases and how to select the right one to increase primer specificity\
  \ is still a challenge. RESULTS: In this study, different from the previous reports\
  \ which used a limited quantity of primers randomly (several or dozen pairs), we\
  \ systematically investigated the effects of mismatch base pairs, mismatch sites\
  \ and SNP types on primer specificity with 2071 primer pairs, which were designed\
  \ based on SNPs from Brassica oleracea 01-88 and 02-12. According to the statistical\
  \ results, we (1) found that the primers designed with SNP (A/T), in which the mismatch\
  \ (CA) in the 3rd nucleotide from the 3' end, had the highest allele-specificity\
  \ (81.9%). This information could be used when designing primers from a large quantity\
  \ of SNP sites; (2) performed the primer design principle which forms the one and\
  \ only best primer for every SNP type. This is never reported in previous studies.\
  \ Additionally, we further identified its availability in rapeseed (Brassica napus\
  \ L.) and sesame (Sesamum indicum). High polymorphism percent (75%) of the designed\
  \ primers indicated it is a general method and can be applied in other species.\
  \ CONCLUSION: The method provided in this study can generate primers more effectively\
  \ for every SNP site compared to other AS-PCR primer design methods. The high allele-specific\
  \ efficiency of the SNP primer allows the feasibility for low- to moderate- throughput\
  \ SNP analyses and is much suitable for gene mapping, map-based cloning, and marker-assisted\
  \ selection in crops."
publication: '*Plant Methods*'
doi: 10.1186/1746-4811-8-34
---
