---
# Documentation: https://wowchemy.com/docs/managing-content/

title: Comparative transcriptome analysis of compatible and incompatible Brassica
  napus - Xanthomonas campestris interactions
subtitle: ''
summary: ''
authors:
- Li Yang
- Chuanji Zhao
- Zetao Bai
- Lingli Yang
- Michael Eric Schranz
- Shengyi Liu
- Klaas Bouwmeester
tags: []
categories: []
date: '2022-01-01'
lastmod: 2022-08-19T18:45:29+08:00
featured: false
draft: false

# Featured image
# To use, add an image named `featured.jpg/png` to your page's folder.
# Focal points: Smart, Center, TopLeft, Top, TopRight, Left, Right, BottomLeft, Bottom, BottomRight.
image:
  caption: ''
  focal_point: ''
  preview_only: false

# Projects (optional).
#   Associate this post with one or more of your projects.
#   Simply enter your project's folder or file name without extension.
#   E.g. `projects = ["internal-project"]` references `content/project/deep-learning/index.md`.
#   Otherwise, set `projects = []`.
projects: []
publishDate: '2022-08-19T10:45:29.500764Z'
publication_types:
- '2'
abstract: Black rot caused by the vascular pathogenic bacterium Xanthomonas campestris
  pv. campestris (Xcc) is widespread in Brassicaceae plants and an infectious disease
  that causes large yield losses in oil seed rape (Brassica napus L.). Improvement
  of resistance through breeding is a crucial strategy to prevent black rot disease
  in B. napus, but presently hampered by insufficient understanding of Xcc-Brassica
  interactions. This study compares two EMS-mutagenized B. napus lines that show contrasting
  resistance levels to their susceptible progenitor. Patterns of differential gene
  expression between these B. napus lines were evaluated at three time points post
  inoculation by comparative RNA-seq analysis. In line with the observed disease phenotypes,
  the susceptible line ZS9mXccS-1 displayed a steady amount of differentially expressed
  genes (DEGs) at different time points of infection, whereas the resistant line ZS9mXccR-1
  displayed a gradual increase in DEGs throughout the course of infection. Weighted
  gene co-expression network analysis (WGCNA) pinpointed multiple defense-related
  hub genes with potential central roles in immunity, including the cell surface receptor
  genes CRK11 and BIR1, and the associated downstream regulatory genes WRKY11 and
  PBL30. KEGG analysis of DEGs belonging to two distinct co-expression modules revealed
  enriched pathways associated with defense, including Ca2+-signalling, receptor-mediated
  immunity, and phytohormone balance. Taken together, our comparative transcriptome
  analysis provides new avenues to unravel the mechanisms underlying black rot resistance
  in B. napus.
publication: '*Frontiers in Plant Science*'
doi: 10.3389/fpls.2022.960874
---
